api/config/routes.rb

```diff
@@ -91,14 +91,7 @@
     resources :states, only: [:index, :show]
 
     resources :taxonomies do
-      member do
-        get :jstree
-      end
-      resources :taxons do
-        member do
-          get :jstree
```

Comment:
@Sinetheta yea if you can at least pull those commits that remove the ruby side out into a separate PR it would be nice.